package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mapbox/vtcomposite",
-  "version": "1.1.0",
+  "version": "1.2.0-dev0",
```

Comment:
because the interface changed in a way that is not backwards compatible, I think this should be v2

Reply:
Fixed in 3711e05918f95ff3494302a0c36e07fe1bdfad08.